src/services/recim/participant.ts

```diff
@@ -0,0 +1,106 @@
+import http from '../http';
+import { Team } from './team';
+
+export type Participant = {
+  Username: string;
+  Email: string;
+  Role: string;
+  Status: string;
+  Notification: ParticipantNotification[];
+};
+
+type PatchParticipantActivity = {
+  ActivityID: number;
+  ActivityPrivType: string;
+  IsFreeAgent: boolean;
+};
+
+type PatchParticipantStatus = {
+  StatusDescription: string;
+  EndDate: string;
+};
+
+type ParticipantStatus = {
+  Username: string;
+  Status: string;
+  StartDate: string;
+  EndDate: string;
+};
+
+type CreatedParticipantStatus = {
+  ID: number;
+  ParticipantID: number;
+  StatusID: number;
+  StartDate: string;
+  EndDate?: string;
+};
+
+type UploadParticipantNotification = {
+  Message: string;
+  EndDate: string;
+};
+
+type ParticipantNotification = {
+  ID: number;
+  Message: string;
+  DispatchDate: string;
+};
+
+type CreatedParticipantNotification = {
+  ID: number;
+  ParticipantID: number;
+  Message: string;
+  EndDate: string;
+  DispatchDate: string;
+};
+
+type CreatedParticipantActivity = {
+  ID: number;
+  ActivityID: number;
+  ParticipantID: number;
+  PrivTypeID: number;
+  isFreeAgent: boolean;
+};
+
+//Participant Routes
+const createParticipant = (ID: number): Promise<Participant> =>
+  http.post(`recim/participants/${ID}`);
```

Comment:
I think this should technically be a PUT operation. Since you already know the unique ID that you want to use (ID), you are 'putting' data in that location. POST operations are for when you want the API to figure out an identifier to use for the object data that you are passing to it.